src/servers/ZoneServer2016/managers/worldobjectmanager.ts

```diff
@@ -61,6 +63,7 @@ function getRandomSkin(itemDefinitionId: number) {
     Items.HAT_BEANIE,
     Items.HAT_CAP,
     Items.HELMET_MOTORCYCLE,
+    Items.KEVLAR_DEFAULT,
   ];
   if (!allowedItems.includes(itemDefinitionId)) return itemDefinitionId;
```

Comment:
Why is this check even here when there is a switch statement that could just use the default clause to early return which is faster?

Reply:
The switch statement can be optimized by V8 to be `O(1)` or `O(log n)` if possible, currently, it is checking twice which is an extra O(n) operation that could be `O(1)` or `O(log n)`.

Reply:
It is also possible that this causes extra heap memory allocations but I don't know how good V8 is at doing stack allocation.